### DISCUSSION TITLE
Migrate in TEST and PROD environment

### DISCUSSION THREAD

**Discussion by kane1997** [prisma/prisma#19093] [Q&A]
In my .env file, I have two Database
`TEST_DATABASE_URL="your_test_database_url"
PRODUCTION_DATABASE_URL="your_production_database_url"`,

I am currently `npx prisma db push` in Test ENV and how to switch to production ENV, how to write the scripts here, How to migrate to production ENV?

**Comment by kane1997** [comment]
Hi I have read about it, in my case for one database URL, I can use npx
prisma db push to test and migrate, but I have two dataBase URL,,, my
question is about how to set these .env values and scripts to execute the
migrate from test to production.
Raphael Etim ***@***.***> 于2023年5月6日周六 00:15写道：

**Reply by ludralph** [reply]
Hi @kane1997 

There is an open [issue](https://github.com/prisma/prisma/issues/2443) to connect to multiple databases. However, there is a [workaround](https://github.com/prisma/prisma/issues/2443#issuecomment-630679118) also included in that issue to connect to multiple databases.

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 
Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Reply by nurul3101** [reply]
We are closing this Discussion due to inactivity. Please open a new Discussion if you have a similar problem 🙏
